### ISSUE TITLE
App not working on iPadOS beta (iOS 13)

### ISSUE THREAD

**Issue by pablopunk** [blinksh/blink#734]
Just downloaded the app on a 2018 iPad and all I get is a black screen

**Comment by pablopunk** [comment]
Tested iOS 13 on iPhone XR and it is working

**Comment by hpetersen** [comment]
Not working in iPadOS (iOS 13) on iPad Pro 11" 3rd Gen.
The app just shows black screen, and closes after a minute or so.

**Comment by yury** [comment]
I have this device, and it is working for me... Must be smth with generating default id_rsa....

**Comment by pablopunk** [comment]
@yury that sounds right, cause I installed it *after* upgrading, so I guess it does need to generate it

**Comment by hpetersen** [comment]
Only the App Store version has this issue. When building from source it's working perfectly on iPadOS 13.

**Comment by pablopunk** [comment]
@hpetersen let's see if I can do that and I'll close the issue 🙌

**Comment by yury** [comment]
@hpetersen I can add you to TF so you can check that.

**Comment by pablopunk** [comment]
@hpetersen I asked for an invitation on twitter today, if u can get it for me that would be awesome

**Comment by ericboehs** [comment]
It would be nice to see the app store version working especially if it's still broke for the public beta. Anyone upgrading to the betas will come looking for an issue in GitHub.

**Comment by hpetersen** [comment]
@yury Would be cool if you could add me to TF. -> hpetersen@me.com

@pablopunk Sorry, I do not have the privilege to add you to TF. Only @yury can do that I guess.

**Comment by yury** [comment]
@hpetersen sent invite.

**Comment by glberen** [comment]
Any way I can get an invite to the TestFlight version?

**Comment by yury** [comment]
@glberen, sure. Email me yury.korolev@gmail.com

**Comment by hpetersen** [comment]
Thank you @yury 👍 

However i still get crashes with the TestFlight version.
I sent you the crash log.

**Comment by glberen** [comment]
Thank you. I’ve got the same issue with the TF version as well. Black screen for several seconds before it dumps out.

**Comment by pablopunk** [comment]
I just compiled it from scratch and it does work 👍

**Comment by thecolorchanges** [comment]
Was wondering if I could try the TestFlight version as well? iPad 11 and iPad 12.9 pro (2018) both crashing on startup.

**Comment by yury** [comment]
Sure, please email me a tyury.korolev@gmail.com

**Comment by pablopunk** [comment]
TLDR: TF version doesn't work yet but compiling the latest code from the repo does work, so I'd say this issue is addressed.
